config.yaml

```diff
@@ -6,6 +6,8 @@ wandb:
 debug:
   env:
     headless: False
+    head_ids: #Which environment IDs to display in pyboy (env_id starts at 0 when serial in debug mode)
```

Comment:
Hrm with this change, headless makes less sense. It can be controlled through head_ids.